dist/src/test/java/io/camunda/application/UnifiedConfigurationIT.java

```diff
@@ -0,0 +1,133 @@
+import io.camunda.configuration.beans.BrokerBasedProperties;
+import io.camunda.exporter.config.ExporterConfiguration;
+import io.camunda.operate.conditions.DatabaseType;
+import io.camunda.operate.property.OperateProperties;
+import io.camunda.tasklist.property.TasklistProperties;
+import io.camunda.zeebe.broker.system.configuration.ExporterCfg;
+import java.util.Map;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.DynamicPropertyRegistry;
+import org.springframework.test.context.DynamicPropertySource;
+import org.springframework.test.context.TestPropertySource;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.utility.DockerImageName;
+
+@SpringBootTest(
+    classes = {
+      StandaloneCamunda.class,
+      // Unified Configuration classes
+      UnifiedConfiguration.class,
+      UnifiedConfigurationHelper.class,
+      TasklistPropertiesOverride.class,
+      OperatePropertiesOverride.class,
+      GatewayBasedPropertiesOverride.class,
+      BrokerBasedPropertiesOverride.class,
+      ActorClockControlledPropertiesOverride.class,
+      IdleStrategyPropertiesOverride.class
+    })
+@TestPropertySource(
+    properties = {
+      // Using opensearch because it's different than the default value
+      "camunda.data.secondary-storage.type=opensearch"
+    })
+@ActiveProfiles({"broker", "tasklist", "operate"})
+public class UnifiedConfigurationIT {
+
+  static final String OPENSEARCH_CONTAINER_NAME = "opensearchproject/opensearch";
+  static final String OPENSEARCH_CONTAINER_VERSION = "2.13.0";
+  static final int OPENSEARCH_DEFAULT_PORT = 9200;
+
+  static GenericContainer<?> opensearchContainer;
+
+  @Autowired UnifiedConfiguration unifiedConfiguration;
+  @Autowired OperateProperties operateProperties;
+  @Autowired TasklistProperties tasklistProperties;
+  @Autowired BrokerBasedProperties brokerBasedProperties;
+
+  @BeforeAll
+  static void setup() {
+    opensearchContainer =
+        new GenericContainer<>(
+                DockerImageName.parse(
+                    OPENSEARCH_CONTAINER_NAME + ":" + OPENSEARCH_CONTAINER_VERSION))
+            .withExposedPorts(OPENSEARCH_DEFAULT_PORT);
+    opensearchContainer.start();
+  }
+
+  @AfterAll
+  static void tearDown() {
+    if (opensearchContainer != null) {
+      opensearchContainer.stop();
+    }
+  }
+
+  @DynamicPropertySource
+  static void registerElasticsearchContainerProperties(DynamicPropertyRegistry registry) {
+    registry.add("camunda.data.secondary-storage.opensearch.url", () -> getOpensearchEndpoint());
+  }
+
+  @Test
+  public void testUnifiedConfigurationLoadsSuccessfully() {
```

Comment:
:x: @buccarel This is not enough. We have to really verify that the application works. Something similar to `io.camunda.it.StandaloneCamundaTest`. The test should verify that we can create an instance and it is visible via search clients, for example. 

Also, I think we can move this test to `io.camunda.it` as well.